pyproject.toml

```
@@ -0,0 +1,97 @@
+[tool.black]
+line-length=119
```

I would vote for 79 line length. 119 - is not readable and too long for me.
@oamg/convert2rhel wdyt?

---

@ZhukovGreen  I just think 79 is too small... in some cases (eg: conditionals) will be weird to read.. I vote between this two: 88 line length as `black` default.

---

It was 80 when I started. Then we decided for 120 about a year ago. So I vote for not getting back to the discussion and stick to the length we have now. I can find the PR where we discussed it if you want.

```suggestion
line-length=120
```

---

Ok, no need to block on this. Applied